controllers/manager/manager.go

```diff
@@ -69,6 +70,11 @@ const (
 	OtherNetworkType = "other"
 )
 
+const (
+	PersonalityActiveController  = "Controller-Active"
```

Comment:
The personality refers controller, worker, storage.
Don't mix the active/standby with personalities